test/comparison-tests/create-and-execute-test.js

```diff
@@ -371,6 +371,8 @@ function getNormalisedFileContent(file, location, test) {
                 .replace(/[\d]+[.][\d]* kB/g, ' A-NUMBER-OF kB')
                 // We also don't want a difference in the number of bytes to fail the build
                 .replace(/ \d+ bytes /g, ' A-NUMBER-OF bytes ')
+                // Sometimes "[built]" is written to output, and sometimes not. This should not fail the build
```

Comment:
Actually having "[built]" is significant I believe.  It indicates whether a module has been recompiled as part of a run.  I believe this is important for the integration tests which have multiple iterations.